ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKStreamTest.java

```diff
@@ -421,33 +413,6 @@ public void testSelectWithExpression() {
 
   }
 
-  @Test
```

Comment:
Also, why did you remove these tests?

Reply:
Because there are QTT tests that cover the same thing.